types/setting.go

```diff
@@ -858,6 +861,16 @@ var (
 		ReadOnly: false,
 		Default:  "false",
 	}
+
+	SettingDefinitionEngineReplicaTimeout = SettingDefinition{
```

Comment:
Remember updating default-setting yaml, and manifests in longhorn/longhorn.

Reply:
Updated. Thank you.